diabetes_regression/training/train.py

```diff
@@ -47,6 +48,12 @@ def train_model(run, data, alpha):
     return reg
 
 
+def get_model_parameters():
+    with open("train.json") as f:
```

Comment:
Feels like we should have a single `config.json` file which could be used to configure such parameters for train or anything other code in the `diabetes_regression` area.

Reply:
I moved config.json to the root folder, but I am limited in tools here to create any utility code (if we don't want to make many changes of course). So, I made it's simple: no serializers, utility code etc. - just open the file and read hardcoded parameter from there. Yes, I modified the structure of the file a bit to satisfy the folder structure.